lib/rucio/web/rest/webpy/v1/replica.py

```diff
@@ -102,6 +102,10 @@ def GET(self, scope, name):
             if 'limit' in params:
                 limit = int(params['limit'][0])
 
+        # Resolve all reasonable protocols when doing metalink for maximum access possibilities
+        if metalink and schemes is None:
+            schemes = ['gsiftp', 'srm', 'root', 'davs', 'https', 'file']
```

Comment:
We have lists of all possible schemes in several places in the code. I wonder if we should replace this with something centrally specified or maybe even something community specific?

Reply:
While I agree a central place makes sense, I don't really see much need for a configurable list as the supported protocols per VO will be automatically resolved through the RSEs anyway?

Reply:
Yes, I was just putting it in the room. A central list of all rucio supported protocols should be sufficient, the actual VO support is derived from the used protocols anyway.